change/office-ui-fabric-react-2020-06-01-14-39-06-commandbar-cache.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "CommandBar: Fixed cacheing",
```

Comment:
```suggestion
  "comment": "CommandBar: Fixed caching",
```

Reply:
the built in caching was not working. Customer doesn't need to do anything. This should be a huge perf gain for anyone using command bar.